src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -58,8 +58,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Mark that this targets file supports package download. -->
     <PackageDownloadSupported>true</PackageDownloadSupported>
     <!-- Flag if the Central package file is enabled -->
-    <DisableCentralPackageVersions Condition="'$(DisableCentralPackageVersions)' == ''">true</DisableCentralPackageVersions>
-    <_CentralPackageVersionsEnabled Condition="'$(DisableCentralPackageVersions)' != 'true' AND '$(CentralPackageVersionsFileImported)' == 'true'">true</_CentralPackageVersionsEnabled>
+    <EnableCentralPackageVersions Condition="'$(EnableCentralPackageVersions)' == ''">false</EnableCentralPackageVersions>
```

Comment:
You don't need this line.

Reply:
Is this opt-in temporary? 

See: https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions#opt-in-central-package-version-management

Reply:
This flag will be disabled until the entire feature is ready.

Reply:
You don't need to set anything to false, empty is fine.

In MSBuild it's a bad pattern to set a default that no one respects. 

Specifically just checking for the 
> `EnableCentralPackageVersions` == 'true' 
is enough.